lib/oidc_old.py

```diff
@@ -48,6 +48,9 @@ def callback(self, fn):
         self._callback_fn = fn
         return fn
 
+    def oidc_providers(self):
```

Comment:
Remove the prefix `oidc_` from this method name - it results in `oidc.oidc_providers`, which is redundant.